scripts/introspection_query.py

```diff
@@ -97,7 +97,7 @@
 
 def print_schema(port, uri, extra_headers):
     conn = http.client.HTTPConnection("localhost", port)
-    headers = {"Content-type": "application/json; charset=uft",
+    headers = {"Content-type": "application/json",
```

Comment:
we should keep `; charset=utf-8` (maybe it was a typo before?)

Reply:
Found out that we didn't need this option for the header. The introspection query wouldn't work with the option.